sonic-net/sonic-buildimage
# kdump support for marvell-arm64

#### Why I did it
To make kdump work in marvell-arm64 platforms

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Enabled kdump-tools download for uboot platforms also, and added crashkernel command line parameters for ac5x hwsku 

#### How to verify it
Verify kernel coredump is created and saved under /var/crash folder.
Also verify show kdump config/files/logging works

#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)
master - 
#### Description for the changelog

#### Link to config_db schema for YANG module changes
NA

#### A picture of a cute animal (not mandatory but encouraged)
